swagger2.0/oic.wk.col.swagger.json

```diff
@@ -525,51 +416,22 @@
       ],
       "type": "object"
     },
-    "oic.oic-link" : {
+    "slinks" : {
+      "type" : "array",
+      "items" : {
+        "$ref": "#/definitions/oic.oic-link"
+      }
+    },
+    "oic.oic-link": {
       "properties": {
-        "anchor": {
-          "description": "This is used to override the context URI e.g. override the URI of the containing collection.",
-          "format": "uri",
-          "maxLength": 256,
-          "type": "string"
-        },
-        "di": {
-          "description": "The Device ID formatted according to IETF RFC 4122.",
-          "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
-          "type": "string"
-        },
-        "eps": {
-          "description": "the Endpoint information of the target Resource",
-          "items": {
-            "properties": {
-              "ep": {
-                "description": "Transport Protocol Suite + Endpoint Locator",
-                "format": "uri",
-                "type": "string"
-              },
-              "pri": {
-                "description": "The priority among multiple Endpoints",
-                "minimum": 1,
-                "type": "integer"
-              }
-            },
-            "type": "object"
-          },
-          "type": "array"
-        },
-        "href": {
-          "description": "This is the target URI, it can be specified as a Relative Reference or fully-qualified URI.",
-          "format": "uri",
-          "maxLength": 256,
-          "type": "string"
-        },
         "if": {
           "description": "The interface set supported by this resource",
           "items": {
             "enum": [
               "oic.if.baseline",
               "oic.if.ll",
               "oic.if.b",
+              "oic.if.lb",
```

Comment:
Why is "oic.if.lb" here?  Also, why is every possible "if" value listed?  It seems like it should only be baseline, ll and b.

Reply:
This is the Link Parameter 'if', not the Resource Property 'if'.  Hence every possible value is here.  Agree oic.if.lb is there for historical reasons and can be deleted.